src/Medulla/Kubernetes/Operator/Model/Database/ValidateSpec.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the Medulla Contributors under one or more agreements.
+// The Medulla Contributors licenses this file to you under the Apache 2.0 license.
+// See the LICENSE file in the project root for more information.
+
+namespace Operator.Model
+{
+
+    /// <summary>
+    /// Spec in Column for validation
+    /// </summary>
+    public struct ValidateSpec
+    {
+        /// <summary>
+        /// matches this RegExp /^[a-z]+$/i
+        /// </summary>
+        public string? Is { get; set; }
+        /// <summary>
+        /// matches this RegExp /^[a-z]+$/i
+        /// </summary>
+        public string? Not { get; set; }
+        /// <summary>
+        /// checks for email format (foo@bar.com)
+        /// </summary>
+        public bool? IsEmail { get; set; }
```

Comment:
Bools have a default value of false, I think this is appropriate for this and the following values? @keithlogan94 if you want to chime in

Reply:
So no need to change it then correct?